packages/guilded-api-typings/src/v1/WS.ts

```diff
@@ -31,7 +31,7 @@ export interface WSMessagePayload {
 }
 
 /** The Guilded WebSocket API operation codes. */
-export enum WSOpCodes {
+export enum WSOpCode {
```

Comment:
I have remembered that typescript increnents enum entries by 1 and start fron 0, i think it would be good to remove the explicit assignment of each op code .